rd-net/Lifetimes/Lifetimes.csproj

```diff
@@ -15,6 +15,8 @@
         <Configurations>Debug;Release;CrossTests</Configurations>
         
         <Platforms>AnyCPU</Platforms>
+        
+        <LangVersion>10</LangVersion>
```

Comment:
This shouldn't be required, since I've enabled this globally via `Directory.Build.props` in `master`.